safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -238,28 +268,37 @@ def accept(self):
     def on_open_input_tool_clicked(self):
         """Autoconnect slot activated when open input tool button is clicked.
         """
+        input_path = self.input_path.text()
+        if not input_path:
+            input_path = os.path.expanduser('~')
```

Comment:
Something that would be very nice to fix for the user experience, it's to follow the last input directory. We should use the QSettings in the `UI` section.
![screen shot 2017-11-13 at 07 43 28](https://user-images.githubusercontent.com/1609292/32712765-954e7770-c846-11e7-8013-b176ec59ad79.png)

I know a lot of users hate when a software doesn't remember your last used folder and bring you to your home directory everytime. Something we should think of when we create new dialog.

Reply:
Yes, I also hate it actually.
Is there a method or widget that do it automatically? Like checking `last` + `widget name or something` in QGIS?

Reply:
QgsFileWidget, but we still need to load/save the last used path.